README.md

```diff
@@ -105,6 +105,19 @@ Bevy is free and open source! All code in this repository is dual-licensed under
 
 at your option. This means you can select the license you prefer! This dual-licensing approach is the de-facto standard in the Rust ecosystem and there are [very good reasons](https://github.com/bevyengine/bevy/issues/2373) to include both.
 
-Unless you explicitly state otherwise, any contribution intentionally submitted
```

Comment:
I think we should leave this here (without any new line breaks). We should move the 

> This means you can select the license you prefer! This dual-licensing approach is the de-facto standard in the Rust ecosystem and there are [very good reasons](https://github.com/bevyengine/bevy/issues/2373) to include both.

section to either above or below.